### PR TITLE
docs: add roles and permissions docs

### PR DESCRIPTION
**Pull request by jorgeepc**
This PR adds the Roles and Permissions docs. It includes:
- Organization roles
- Environment roles
- Adding organization members by email
- Adding organization members by invite link

## Changes

- roles and permissions docs

## Fixes

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Screenshot

screencapture-localhost-3000-concepts-roles-and-permissions-2023-11-30-16_29_18 (1)

### CODE REVIEW

File: docs/docs/concepts/roles-and-permissions.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  - permissions
+  - members
+image: https://res.cloudinary.com/djwdcmwdz/image/upload/v1698686403/docs/Blog_Thumbnail_14_rsvkmo.jpg
+---
+
+In this section, you will learn everything you need to manage your organization members effectively.
+
+Tracetest distinguishes between different roles to help manage organization members' access levels and permissions. These roles are categorized into two groups: organization level and environment level roles.
+
+## Organization roles
```
Comment (jfermi) [inline docs/docs/concepts/roles-and-permissions.mdx:22]:
```suggestion
## Organization Roles
```

File: docs/docs/concepts/roles-and-permissions.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+- __Member__: Members are users who only need to view data within the organization, but do not need to edit or modify any setting. Members are typically used for adding users to specific environments with a determined level of access.
+
+- __Biller__: Billers are responsible for managing the billing and payment information for the organization. The billing role also gets read-only access to the organization.
+
+## Environment roles
+
+Environment level roles provide fine-grained control and access to specific environments within an organization. These roles are assigned to individuals and are restricted to the environments they are assigned to. Environment roles include:
+
+- __Admin__: Organization owners and admins inherently act as environment administrators for every environment. Environment administrators can create any test related resource, and manage all environment settings.
```
Comment (jfermi) [inline docs/docs/concepts/roles-and-permissions.mdx:38]:
```suggestion
- __Admin__: Organization owners and admins inherently act as environment administrators for every environment. Environment administrators can create any test related resource and manage all environment settings.
```

File: docs/docs/concepts/roles-and-permissions.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+- __Engineer__: Engineers have control of the environment resources related to tests creation and execution. However, environment engineers can not manage the environment settings.
+
+- __Runner__: Runners have read-only access to a specific environment. Environment runners can only re-run tests that already exist in the environment.
+
+## Managing organization members
```
Comment (jfermi) [inline docs/docs/concepts/roles-and-permissions.mdx:44]:
```suggestion
## Managing Organization Members
```

File: docs/docs/concepts/roles-and-permissions.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+As the organization owner/admin, you have the ability to manage your organization's composition and the roles of its members. These role assignments define the access level each member has across environments within the organization's scope.
+
+### Adding organization members by email
```
Comment (jfermi) [inline docs/docs/concepts/roles-and-permissions.mdx:48]:
```suggestion
### Adding Organization Members by Email
```

File: docs/docs/concepts/roles-and-permissions.mdx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+### Adding organization members by invite link
+
+Organization owners/admins can also share an invite link with others to allow them to join the organization and environments without needing to be invited individually. To generate an invite link:
+
+From the organization dashboard, select the `Invites` tab and the `By Invite Link` option.
+
+![Select invites tab](../img/members-invite-01.png)
+
+Enter a name for the invite link. The invite link will give users a `Member` role within the organization, however you can select the environments and the corresponding role for them. Optionally, you can add a URL that will be used to redirect the user once the invitation is accepted. Finally click the `Create Invite Link` button.
```
Comment (jfermi) [inline docs/docs/concepts/roles-and-permissions.mdx:74]:
```suggestion
Enter a name for the invite link. The invite link will give users a `Member` role within the organization, additionally, you can select the environments and the corresponding role for them. Optionally, you can add a URL that will be used to redirect the user once the invitation is accepted. Finally click the `Create Invite Link` button.
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by kdhamric — APPROVED** [review]
Really nice!

**Pull request merged**